qiskit/optimization/converters/linear_equality_to_penalty.py

```diff
@@ -112,3 +122,39 @@ def encode(self, op: QuadraticProgram, penalty_factor: float = 1e5,
             self._dst.maximize(offset, linear, quadratic)
 
         return self._dst
+
+    def _auto_define_penalty(self, penalty_factor: float = 1e5) -> float:
+        """Automatically define the penalty coefficient.
+
+        This returns (upper bound - lower bound + 1) of the objective function.
```

Comment:
Is this text needed `This returns (upper bound - lower bound + 1) of the objective function.` There is the Returns section below and this text and that differ in what they say this returns. The Returns text seems more plausible